### DISCUSSION TITLE
react-native-webview new architecture adoption

### DISCUSSION THREAD

**Discussion by Titozzz** [reactwg/react-native-new-architecture#19] [Libraries]
Hi there, just a FYI, I'm going to try to migrate [react-native-webview](https://github.com/react-native-webview/react-native-webview) to the new arch 😃

**Comment by cipolleschi** [comment]
Hi @Titozzz! How is it going? Are there any blockers to the migration?

**Comment by Titozzz** [comment]
I got android fully working old and new and iOS is on the way but time can be an issue, also I was waiting on autolinking and 0.70.

If you want to review what I've done so far, that could probably help too

**Reply by cipolleschi** [reply]
Sure, do you have a PR?

**Reply by Titozzz** [reply]
will create the android PR and let you know :)

**Reply by Titozzz** [reply]
You have the WIP directory here : https://github.com/Titozzz/test-webview

**Reply by Titozzz** [reply]
https://github.com/react-native-webview/react-native-webview/pull/2635 PR is here, android is good old/new arch. Last blocker is tests with RNTA but code is working fine.
